imagersite/imagersite/views.py

```diff
@@ -0,0 +1,21 @@
+# from django.http import HttpResponse
+from django.shortcuts import render
+from django.views.generic import TemplateView
+# from django.template import loader
+
+
+def home_view(request):
+    context = {'foo': 'name', 'bar': 'num'}
+    return render(request, 'home.html', context=context)
+
+
+def test_view(request, num=0, name=""):
+    context = {'foo': num, 'bar': name}
+    return render(request, 'home.html', context=context)
+
+
+class ClassView(TemplateView):
+    template_name = 'home.html'
+
+    def get_context_data(self, num=0, name=''):
+        return {'foo': num, 'bar': name}
```

Comment:
This entire file is questionable.  Just because we played in here during class today does not mean that it's the kind of thing you should actually have around.  Same goes for the `hello.html` template above.  What does it do in the context of *your project*?

We do exercises in class to explore the APIs you'll use to finish up your work for the night.  Sometimes, that work is useful to your implementation directly.  Sometimes it isn't.  I want to you *actively engage* in thinking about the exercises and whether they belong in your actual code submission.  That active engagement is how you learn.